src/deepsparse/transformers/engines/nl_decoder_engine.py

```diff
@@ -202,10 +201,7 @@ def __call__(
         else:
             logits = out[0]
 
-        # select batch idx 0, batch is always 1
-        token = self.generate_token(logits=logits[0, -1, :])
-
-        return token, logits
+        return logits
```

Comment:
Please update the return type of this function, given that it no longer outputs logits